tests/tests_default.yml

```diff
@@ -2,6 +2,7 @@
 ---
 - name: Ensure that the role runs with default parameters
   hosts: all
+  become: yes
```

Comment:
is this really needed?

Reply:
Currently all the tests are using that. I've added it just for consistency. In previous PRs @pcahyna asked if we could remove it from all tests but @amore17 said it's been useful for her (for me it's useful as well). 

Basically we run that in machines where we can connect using a non-root user with sudo rights. If we remove it from playbooks I'll have to add the become to the inventories instead of just running tests with `-i ip_address,`. Since adding that to tests it's not harmful I though it was ok to keep.

If you prefer that I keep  `tests/tests_default.yml` as it is I'm fine with that to.

Reply:
re: become - It is ok to leave it - I'll note that there are a few of the older roles that still use `become: true` in the tests - storage, selinux, logging - other than certificate, none of the other roles use `become`.

> Basically we run that in machines where we can connect using a non-root user with sudo rights

What machines are those?  How are you provisioning those machines?

> If we remove it from playbooks I'll have to add the become to the inventories instead of just running tests with -i ip_address,.

Why?  You can't run tests using `ansible-playbook --become .... tests/tests_xxx.yml`?

Reply:
> What machines are those? How are you provisioning those machines?
In my case I have a script that provision machines using virt-install, create a user with sudo rights, add my public key, creates a snapshot, installs IPA and create another snapshot. I usually use this machine while developing.

@amore17 can correct me if I'm wrong but I suppose she uses remote servers in an internal OpenStack environment (using beaker or idm-CI to provision).

> What machines are those? How are you provisioning those machines?
We probably can! I just didn't know about the parameter :)

Reply:
I'm pushed a rebase without it and I'll create another PR removing it from everywhere else.